android/java/org/chromium/chrome/browser/vpn/BraveVpnConfirmDialogFragment.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.chromium.chrome.browser.vpn;
+
+import android.animation.Animator;
+import android.content.DialogInterface;
+import android.content.res.Configuration;
+import android.os.Bundle;
+import android.util.DisplayMetrics;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.WindowManager;
+import android.widget.LinearLayout;
+
+import androidx.fragment.app.DialogFragment;
+
+import com.airbnb.lottie.LottieAnimationView;
+
+import org.chromium.chrome.R;
+import org.chromium.chrome.browser.util.ConfigurationUtils;
+import org.chromium.ui.base.DeviceFormFactor;
+
+public class BraveVpnConfirmDialogFragment extends DialogFragment {
```

Comment:
This class and the class `BraveVpnCalloutDialogFragment` have  completely equal methods 
`onConfigurationChanged`, `onResume()`, `setDialogParams()`  .

If we will have more of similar classes, maybe it would be worth to pull the equal methods to the base class?

Reply:
awesome, thanks